parcels/particlefile.py

```diff
@@ -234,6 +234,11 @@ def convert_pset_to_dict(self, pset, time, deleted_only=False):
             else:
                 if deleted_only:
                     pset_towrite = pset
+                # == commented due to git rebase to master 27 02 2020 == #
+                # elif pset[0].dt > 0:
+                #     pset_towrite = [p for p in pset if time - p.dt/2 <= p.time < time + p.dt and np.isfinite(p.id)]
+                # else:
+                #     pset_towrite = [p for p in pset if time + p.dt < p.time <= time - p.dt/2 and np.isfinite(p.id)]
```

Comment:
Can this commented code be removed?

Reply:
On that one, I am unsure. it"s in there since 3 weeks in the master cause I wanted to first test the patch with our our other applications. As we still have little feedback on the working of recent changes, I'd still rather keep it in until we can really confirm that the patch here works.

We can remove it, but I wouldn't do it right now but rather in a few weeks.

Reply:
This will be changed anyway when we merge #678